src/components/SigningFlow.tsx

```diff
@@ -93,7 +92,9 @@ export default function () {
   const { title, subTitle } = SigningStates[flowState]
 
   return error ? (
-    <ErrorBlock
+    <StatusBlock
+      error
+      loadingText=""
```

Comment:
How about making `loadingText` an optional argument?

Reply:
Oh it's optional already, simply remove this

Reply:
Already did this 🙂